output/paper/TemplateTex.blg

```diff
@@ -0,0 +1,46 @@
+This is BibTeX, Version 0.99d (TeX Live 2021)
```

Comment:
Suggest to add .BLG to .gitignore

Reply:
@jmshapir Just wanted to highlight `.blg` is the BibTeX log file, in case it makes sense to keep the bibliography log.

Reply:
Thanks @mcaceresb! For now my instinct is to ignore the various LaTeX/BibTeX-generated logs and keep the sconscript.log. Users can still have a local copy of LaTeX/BibTeX logs for debugging.

Reply:
@jmshapir Added to gitignore in 493caac.